assets/src/components/v2/disruption_diagram/disruption_diagram.tsx

```diff
@@ -300,144 +248,65 @@ interface EndSlotComponentProps {
   isCurrentStop: boolean;
   isAffected: boolean;
   effect: Effect;
-}
-
-interface FirstSlotComponentProps extends EndSlotComponentProps {
   spaceBetween: number;
+  isLeftSide: boolean;
+  x: number
 }
 
-const FirstSlotComponent: ComponentType<FirstSlotComponentProps> = ({
+const EndSlotComponent: ComponentType<EndSlotComponentProps> = ({
   slot,
   line,
-  spaceBetween,
+  isCurrentStop,
   isAffected,
   effect,
-  isCurrentStop,
+  spaceBetween,
+  isLeftSide,
+  x
 }) => {
   let icon;
   if (slot.type === "arrow") {
-    icon = (
-      <path
-        className={classWithModifier("end-slot__arrow", line)}
-        d="M35 0V24L19.554 24C19.1915 24 18.8358 23.9015 18.525 23.715L1.85831 13.715C0.563633 12.9382 0.563633 11.0618 1.85831 10.285L18.525 0.285015C18.8358 0.0985165 19.1915 0 19.554 0L35 0Z"
-        fill={line}
-      />
-    );
+    icon = isLeftSide ?
+      // Not sure where the 22 comes from
+      <g transform={`translate(-22 0)`}><LeftArrowEndpoint className={classWithModifier("end-slot__arrow", line)} /></g>
```

Comment:
Hmm weird, so after the refactor the left-facing arrow was 22px too far to the right for some reason?

Reply:
😕 yeahhh... Yeah the comment could be better, lol. But the x translate is needed to make it look correct